abci/types/types.pb.go

```diff
@@ -1,7 +1,52 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: abci/types/types.proto
 
-//nolint
+/*
```

Comment:
We need to lock these binary dependencies down or this stuff will keep happening.

Reply:
https://github.com/tendermint/tendermint/issues/2238